src/components/ReportActionItem/ReportPreview.tsx

```diff
@@ -511,6 +508,95 @@ function ReportPreview({
         Navigation.navigate(ROUTES.REPORT_WITH_ID.getRoute(iouReportID));
     }, [iouReportID]);
 
+    const reportPreviewAction = useMemo(() => {
+        if (!iouReport || !policy || !transactions) {
```

Comment:
This condition won't work as you intent because in case of 1:1 IOU reports (those without workspace) there will be no policy and it's intended. So please update this condition to `if (!iouReport) {` and modify `getReportPreviewAction` so it accepts policy as optional argument. You also don't need `!transations` part because `transactions` are always defined as empty array.